app/views/users/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Welcome to BudgetApp</h1>
+
+<h2>If you are already a member please signin above</h2>
+<h2>To become a member please complete your details below</h2>
```

Comment:
Should these be `h2` or would you consider styling them bigger with stylesheets?